app/assets/javascripts/kaui/kaui.js

```diff
@@ -138,4 +138,43 @@ jQuery(document).ready(function ($) {
         }
     }
 
+    /*
+     *  Validate external key
+     */
+    const VALIDATE_EXTERNAL_KEY = {
+        account: { url: '/kaui/accounts/validate_external_key', invalid_msg_class_name: '.account_external_key_invalid_msg' },
```

Comment:
This won't work when mounted in the standalone war (`kaui` engine is mounted at the root path).

We can use [js-routes](https://github.com/railsware/js-routes) though (see [this](https://github.com/killbill/killbill-admin-ui/blob/30c60ef14925a1d2e6c3aa63b75589f258f310d9/app/views/kaui/accounts/_billing_info.html.erb#L131) for an example).